src/current/_includes/releases/v23.1/v23.1.8.md

```diff
@@ -0,0 +1,33 @@
+## v23.1.8
+
+Release Date: August 8, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-8-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Azure Blob Storage credentials for implicit authentication](../v23.1/cloud-storage-authentication.html?filters=azure#azure-blob-storage-implicit-authentication) can now be stored in a file with the following YAML format:
+
+    ```
+    azure_tenant_id: MY_TENANT_ID,
+    azure_client_id: MY_CLIENT_ID,
+    azure_client_secret: MY_CLIENT_SECRET
+    ```
+    Set the environment variable `COCKROACH_AZURE_APPLICATION_CREDENTIALS_FILE` with the path of the file. The implicit authentication method now first looks for credentials in the file before proceeding to look at the rest of the chain specified in the DefaultAzureCredential class, such as environment variables and Managed Identity. These credentials can also be used for [Key Management Service (KMS)](../v23.1/take-and-restore-encrypted-backups.html#use-key-management-service). [#106688][#106688]
+
+<h3 id="v23-1-8-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where [`DROP INDEX ... CASCADE`](../v23.1/drop-index.html) could drop unrelated [foreign key constraints](../v23.1/foreign-key.html). For example, when `DROP INDEX ... CASCADE` was executed on a unique index for a table A, unrelated outbound foreign key constraints on a table B would be dropped, if table B had a foreign key reference to table A. [#107793][#107793]
+- Fixed a bug that would result in corruption of the [encryption at rest](../v23.1/security-reference/encryption.html#encryption-at-rest) state which would cause the loss of a node after restart. If multiple nodes encountered this bug at the same time during roll out, this could result in loss of [quorum](../v23.1/architecture/replication-layer.html#overview). See Technical Advisory 106617 for more information. [#107277][#107277]
```

Comment:
```suggestion
- Fixed a bug that would result in corruption of [encrypted data at rest on a cluster node](../v23.1/security-reference/encryption.html#encryption-at-rest). If a node with this corrupted state was restarted, it could fail to rejoin the cluster.  If multiple nodes encountered this bug at the same time during rollout, the cluster could lose [quorum](../v23.1/architecture/replication-layer.html#overview). Refer to Technical Advisory 106617 for more information. [#107277][#107277]
```
Consider removing the last sentence until the TA is available. I wonder if you could say something like "This note will be updated when more information is available."

The note is a bit unsatisfying -- As a person considering the upgrade, I don't have any sense of what the bug was or whether I'm likely to run into it. Can we provide the versions that are impacted by the bug? Do we want to tell them to contact support if a node fails to rejoin the cluster upon restart?

Reply:
I made the modifications suggested and changed the last sentence to:
> This note will be updated when more information is available for Technical Advisory 106617.